packages/cli/src/commands/exec.js

```diff
@@ -45,21 +46,18 @@ export const handler = async (args) => {
   const scriptPath = path.join(getPaths().scripts, name)
 
   // Import babel config for running script
-  babelRequireHook({
-    extends: path.join(getPaths().api.base, '.babelrc.js'),
-    extensions: ['.js', '.ts'],
-    plugins: [
+  registerApiSideBabelHook({
+    additionalPlugins: [
       [
         'babel-plugin-module-resolver',
         {
           alias: {
             $api: getPaths().api.base,
           },
         },
+        'exec-$api-module-resolver',
       ],
     ],
```

Comment:
Mind pointng me to the plugin please? Didn't realise you added it in 😃

Reply:
Here you go! I really struggled with the other one:

https://github.com/redwoodjs/redwood/blob/8e3f9c6ff1a73419791ff885fe171f84d5db05ec/packages/internal/src/build/api.ts#L88-L94

Reply:
It does not support multiple alias, just "srcAbsPath", but it makes the files relative in the resulting ast.

Reply:
Oh, I guess in this case it _WOULD_ not work. Sorry, super tired, but for pre-render it would work

Reply:
Ah.... I think we should leave this as is for now in that case. Because one of the changes I made a while ago was to support user defined aliases (like in Burnsy's project).

What do you think of leaving this for now (to make it non-breaking) - and switch everything over and remove babel-plugin-module-resolver once we get to pre-transpiling web?